src/main/scala/tech/cryptonomic/conseil/tezos/ServiceOperations.scala

```diff
@@ -0,0 +1,63 @@
+package tech.cryptonomic.conseil.tezos
+
+import com.typesafe.config.Config
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.collection.JavaConverters._
+
+
+object ServiceOperations {
```

Comment:
As in previous comments, I would go for more descriptive names, e.g. `PlatformDiscoveryOperations`

Reply:
changed